File: PokemonShopping.Application/DTOs/UserDto.cs
using PokemonShopping.Domain.Models;

namespace PokemonShopping.Application.DTOs
{
    public class UserDto
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Rol { get; set; }
        public List<ShoppingCartDTO> ShoppingCarts { get; set; }
    }
}

File: UI/pokemonShopping/src/app/modules/api/models/user-dto.ts
/* tslint:disable */
/* eslint-disable */
import { ShoppingCartDto } from './shopping-cart-dto';
export interface UserDto {
  email?: null | string;
  id?: string;
  name?: null | string;
  rol?: null | string;
  shoppingCarts?: null | Array<ShoppingCartDto>;
}
